src/helpers.js

```diff
@@ -121,3 +122,33 @@ export const populateConnectedColumn = (connected) => {
     </Popover>
   );
 };
+
+const MAX_RHEL_AI_VERSION = 1.5; // TODO: How to get this dynamically?
+export const populateRHELAIColumn = (system_profile) => {
+  const bootcImage = system_profile?.bootc_status?.booted?.image;
+  if (!bootcImage || !bootcImage.includes('/rhelai')) {
+    return 'N/A';
+  }
+  const rhelaiVersionMatch = bootcImage.match(/:([0-9.-]+)$/);
+  if (!rhelaiVersionMatch) {
+    return 'Unknown';
+  }
+  const rhelaiVersion = rhelaiVersionMatch[1];
+  return parseFloat(rhelaiVersion) < MAX_RHEL_AI_VERSION ? (
```

Comment:
Bold of you to assume that version numbers will be parseable as floats :-D

Reply:
😀 It might at some point be worth to use the `semver`[0] module for this. 

[0] https://www.npmjs.com/package/semver

Reply:
Well, version numbers won't make it past the regex if they aren't 'floatish'.  Also parseFloat is very forgiving.  It will ignore parts of the string that aren't 'floatish', eg '1.4.1' parses to 1.4, '1.1-123' parses to 1.1.  But I will have a look at semver.